src/lib/layers/editable-geojson-layer.js

```diff
@@ -147,11 +156,13 @@ export default class EditableGeoJsonLayer extends EditableLayer {
   }
 
   isFeatureSelected(feature: Object) {
-    if (!this.props.data) {
+    // TODO: Upgrade deck to include https://github.com/uber/deck.gl/commit/40ff66ed
+    // This will be buggy until then
+    if (!this.props.data || !this.props.selectedFeatureIndexes.length) {
```

Comment:
check `!this.props.selectedFeatureIndexes` before `!this.props.selectedFeatureIndexes.length`